client/components/command-pallette/README.md

```diff
@@ -0,0 +1,33 @@
+# Command Palette
+
+The Command Palette is a React component used to display a text input used to search and filter a list of commands. It can be accessed in the Calypso using the shortcut `cmd+k`.
+
+The command palette is inspired by [Core](https://github.com/WordPress/gutenberg/blob/trunk/packages/commands/README.md) implementation used in the Gutenberg editor.
+
+## Add a command
+
+In order to add a new command to the palette, you can add them in `useCommandsArrayWpcom` or create a new hook and include it to `useCommandPallette` hooks.
+
+## Command properties
+
+Every command has a set of properties that can be used to customize its behavior. The following properties are available:
+
+- `name`: Type `string`, used as identifier for the command. It's ex
+- `label`: Type `string`, used as the label for the command. Visible in the command palette.
+- `subLabel`?: Type `string`, used as a second line for the command. Visible in the command palette.
+- `searchLabel`?: Type `string`, used to match the user search. It's not visible for the user. We recommend to use `_x()` to add context to the translators.
+- `callback`: Type `function`, used to execute the command when the user clicks on it or press enter on that command.
+- `context`?: Type `string[]`, a list of paths used to display the command by default.
```

Comment:
I feel we could clarify it more. I'm not sure if it's the best wording, though: 

```suggestion
- `context`?: Type `string[]`, a list of URL paths for which the command should be displayed.
```

Reply:
Changed https://github.com/Automattic/wp-calypso/pull/84591/commits/44a1933d989bb36da1e6b51b2edca86642d7e73b